cloudmock/aws/mockautoscaling/group.go

```diff
@@ -58,13 +58,13 @@ func (m *MockAutoscaling) CreateAutoScalingGroup(input *autoscaling.CreateAutoSc
 		DefaultCooldown:      input.DefaultCooldown,
 		DesiredCapacity:      input.DesiredCapacity,
 		// EnabledMetrics:          input.EnabledMetrics,
-		HealthCheckGracePeriod:  input.HealthCheckGracePeriod,
-		HealthCheckType:         input.HealthCheckType,
-		Instances:               []*autoscaling.Instance{},
-		LaunchConfigurationName: input.LaunchConfigurationName,
-		LoadBalancerNames:       input.LoadBalancerNames,
-		MaxSize:                 input.MaxSize,
-		MinSize:                 input.MinSize,
+		HealthCheckGracePeriod:           input.HealthCheckGracePeriod,
+		HealthCheckType:                  input.HealthCheckType,
+		Instances:                        []*autoscaling.Instance{},
+		LaunchConfigurationName:          input.LaunchConfigurationName,
+		LoadBalancerNames:                input.LoadBalancerNames,
+		MaxSize:                          input.MaxSize,
+		MinSize:                          input.MinSize,
```

Comment:
FYI go 1.11 changed gofmt, and we're still validating against go 1.10 gofmt rules (until k8s/kops 1.13 which moves to 1.11)

Reply:
@justinsb Thanks, fixed.